### PR TITLE
CHANGELOG: update change log for 0.2.1 release

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+
+### Bug Fixes
+ - Fix verification of route rules with 2+ route tables
+ - Support reapply on IPv6 changes
+ - Fix disable IP stack after enabled DHCP
+ - Fix MAC address pattern to support bigger addresses
+ - Report missing OVS NM plugin on profile activation
+ - Ignore existing unmanaged and unknown type bridge ports
+ - Fix bridge type interface state parsing
+ - Assume removed interface as success
```
Comment (tyll) [inline CHANGELOG]: This needs a little bit more context or more of an overview, for example: `Fix success check after interface deletion`

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+
+### New Features
+ - Support SR-IOV VFs configuration
+ - Introduce libnmstate.__version__
+
+### Bug Fixes
+ - Fix verification of route rules with 2+ route tables
+ - Support reapply on IPv6 changes
+ - Fix disable IP stack after enabled DHCP
+ - Fix MAC address pattern to support bigger addresses
```
Comment (EdDev) [inline CHANGELOG]: bigger ==> infiniband ?

File: CHANGELOG [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+ - Report missing OVS NM plugin on profile activation
+ - Ignore existing unmanaged and unknown type bridge ports
+ - Fix bridge type interface state parsing
+ - Assume removed interface as success
+ - Get the SR-IOV total-vfs from sysfs instead from NM configuration
+
```
Comment (EdDev) [inline CHANGELOG]: I'm missing the fix for OVS internal-interface mac-address.
`Read mac-address from sysfs when missing from NM (ovs-interface)`

File: CHANGELOG (hunk 1 above)
Comment (ffmancera) [reply CHANGELOG]: Done! Thanks!

File: CHANGELOG (hunk 2 above)
Comment (ffmancera) [reply CHANGELOG]: Yes, done!

File: CHANGELOG (hunk 3 above)
Comment (ffmancera) [reply CHANGELOG]: Oops, added thanks!

**Pull request merged**